lmctl/cli/commands/permissiontype.py

```diff
@@ -0,0 +1,42 @@
+import click
+from .utils import TNCOCommandBuilder, Identity, Identifier, pass_io
+from .actions import get, retry, rollback, cancel
+from lmctl.client import TNCOClient
+from lmctl.cli.format import Column
+from typing import List
+from lmctl.cli.io import IOController
+
+__all__ = (
+    'get_permissiontype',
+)
+
+from ..arguments import set_param_option
+
+tnco_builder = TNCOCommandBuilder(
+    singular='permissiontype',
+    plural='permissiontypes',
+    display_name='PermissionType'
+)
+
+id_arg = Identifier(param_name='id')
+
+default_columns = [
+    Column('name', header='Name'),
+    Column('description', header='Description'),
+]
+
+@tnco_builder.make_get_command(
+    identifiers=[id_arg],
```

Comment:
Do permission types have IDs? Or is the name used to identify them?

Reply:
names are used to identify the permission types. Addressed.